sandboxes/input/stats_monitor.lua

```diff
@@ -0,0 +1,64 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Hindsight plugins.tsv reader
+
+Reads in the plugins.tsv file to enable plugin monitors to be written.
+
+## Sample Configuration
+```lua
+filename = "plugin_stats.lua"
+ticker_interval = 60
+preserve_data = false
+```
+--]]
+require "io"
+local l = require "lpeg"
+l.locale(l)
+
+local eol = l.P"\n"
+local tab = l.P"\t"
+local header = (1 - eol)^1 * eol
+local plugin = l.C((1 - tab)^1)
+local number = l.digit^1 / tonumber
+local row = l.Cg(plugin * l.Ct((tab * number)^1)) * eol
+local grammar = header * l.Cf(l.Ct"" * row^0, rawset) * -1
+local delta = {}
+local msg = {
+    Type = "hindsight.plugins",
+    Payload = "",
+    Fields = {delta = delta}
```

Comment:
Add name for documentation purposes also the initial delta table is never used it is just gc'ed so it can be left nil.